engines/kgs-bot/vit-bot.yaml

```diff
@@ -0,0 +1,36 @@
+# To launch run the following command from the directory containing this file:
+#     kubectl create -f vit-bot.yaml
+# Make sure to set the credentials of the bot accordingly.
+# Only one copy of the bot can run at a time on one account!
+# You can set the room the bot is in by manually logging in as the bot on KGS,
+# and navigating to the room you want the bot to be in. When the bot reconnects
+# it will join the room you were in.
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: vit-kgs-bot
+  labels:
+    kueue.x-k8s.io/queue-name: farai
+spec:
+  suspend: true # The job must start suspended kueue will start it when it is admitted
+  backoffLimit: 9001
+  template:
+    spec:
+      priorityClassName: normal-batch
```

Comment:
Make this `high-batch` as well? Then it's unlikely to get pre-empted

Reply:
Ah I purposely made the non-pro bot normal-batch so that it could get pre-empted by more important paper experiments, since a few extra games (we have 1000 games now) won't make too much of a difference to our NeurIPS submission.